examples/platforms/efr32/radio.c

```diff
@@ -177,6 +181,12 @@ static RAIL_Handle_t efr32RailConfigInit(efr32BandConfig *aBandConfig)
 
     handle = RAIL_Init(&aBandConfig->mRailConfig, NULL);
     assert(handle != NULL);
+    gRailHandle = handle;
+
+    CMU_ClockEnable(cmuClock_PRS, true);
+    RTCDRV_Init();
```

Comment:
`RTCDRV_Init()` only needs to be called once? So could it be called in `efr32RadioInit()` instead? Also, it could return an error?